src/browser/views/options.jade

```diff
@@ -9,37 +9,85 @@ html
       padding: 2px;
       min-width: 380px;
     }
-    .input {
-      margin-bottom: 10px;
-    }
-    .caption {
-      margin-right: 5px;
-      width: 115px;
-      display: inline-block;
-      vertical-align: top;
-      padding-top: 3px;
-    }
-    .comment {
-      color: #999;
-      margin-left: 5px;
-    }
-    input[type="text"] {
-      width: 25px;
+
+    .option-group {
+      /* Reset the default fieldset styles */
+      margin: initial;
+      border: initial;
+      padding: initial;
+    }
+
+    .option-group + .option-group {
+      margin-top: 30px;
+    }
+
+    .option-group__title {
+      /* Reset the default legend styles */
+      margin: initial;
+      padding: initial;
+
+      margin-bottom: 8px;
       font-weight: bold;
-      text-align: center;
+      font-size: 30px;
+    }
+
+    .option + .option {
+      margin-top: 5px;
+    }
+
+    .option__textarea {
+      margin-top: 2px;
+      width: 300px;
+      min-height: 50px;
+    }
+
+    .option__hint {
+      margin-top: 2px;
+      font-size: 10px;
     }
-    input[type="checkbox"] {
-      width: 24px;
-      height: 20px;
+
+    .option__textarea + .option__hint {
+      margin-top: -2px;
+    }
+
+    /* Checkbox and radio styling */
+    .option_type_checkbox .option__element,
+    .option_type_radio .option__element {
+      vertical-align: bottom;
+    }
+
+    .option_type_checkbox .option__label,
+    .option_type_radio .option__label {
+      margin-left: 4px;
+    }
+
+    .option_type_checkbox .option__textarea,
+    .option_type_checkbox .option__hint,
+    .option_type_radio .option__textarea,
+    .option_type_radio .option__hint {
+      margin-left: 20px;
+    }
+
+
+    /* Checkbox styling */
+    .option_type_checkbox .option__element {
+      /* Radio button in Chrome is 2px narrower than the checkbox.
```

Comment:
No. We'll do it as a separate PR when we'll be ready for that. It will look different in FF for sure, as we're including [the Chrome css file here](https://github.com/zalmoxisus/redux-devtools-extension/blob/master/src/browser/extension/manifest.json#L49).